public/css/index.css

```diff
@@ -0,0 +1,13 @@
+
+#canvas {
+  border: 1px solid black;
+}
+#palette button:nth-child(1) {
```

Comment:
There's probably a nicer way to select individual colour buttons here. Suggestions welcome!